src/cogs/rules.py

```diff
@@ -45,7 +45,8 @@ async def snipe(self, ctx, number = None):
         user = str(message.author)
         ts = message.created_at.isoformat(" ")
```

Comment:
```suggestion
        ts = message.created_at.isoformat(" ", "seconds")
```

Now that we're at it